castervoice/rules/apps/mouse_grids/gridsudoku.py

```diff
@@ -0,0 +1,105 @@
+from castervoice.lib.ctrl.mgr.rule_details import RuleDetails
```

Comment:
All right have changed it to a Mappingrule as CCR app requires an executable. (I need to do a little research on that front)

Here is the full list of imports.
```
import time

from dragonfly import Function, Choice, MappingRule

from castervoice.lib.actions import Mouse

from castervoice.lib import control, settings
from castervoice.rules.ccr.standard import SymbolSpecs
from castervoice.lib.merge.additions import IntegerRefST
from castervoice.lib.merge.state.short import R

from castervoice.lib.ctrl.mgr.rule_details import RuleDetails
from castervoice.asynch.mouse import grids
```